docs/concepts/events.md

```diff
@@ -116,13 +116,27 @@ Some events implement the `ICancellableEvent` interface. These events can be can
 
 Event handlers can opt to explicitly receive cancelled events. This is done by setting the `receiveCanceled` boolean parameter in `IEventBus#addListener` (or `@SubscribeEvent`, depending on your way of attaching the event handlers) to true.
 
-### Results
+### TriStates and Results
 
-Some events have a `Result`. A `Result` can be one of three things: `DENY` which stops the event, `ALLOW` which force-runs the event, and `DEFAULT` which uses the Vanilla behavior. The result of an event can be set by calling `Event#setResult`. Not all events have results; an event with a result will be annotated with `@HasResult`.
+Some events have three potential return states represented by `TriState`, or a `Result` enum directly on the event class. The return states can either cancel the event outcome (`TriState#FALSE`), force the event outcome to happen (`TriState#TRUE`), or execute default Vanilla behavior (`TriState#DEFAULT`).
```

Comment:
I'm not sure that "forcing the event outcome" is quite the right description for `TriState`. It wouldn't fit `setUseBlock()` and `setUseItem()` for example I think. :D

It is more about overriding some checks to force some action to happen or not happen. (<- vague sentence :D )

Reply:
I reworded it to apply to the action rather than an 'event outcome'